index.html

```diff
@@ -0,0 +1,209 @@
+          <h2>Multi-post <br>Story</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt="" /></li>
+            <li><img src="images/css.png" alt=""/></li>
+            <li><img src="images/javaScript.png" alt=""/></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt="" />
+        </div>
+      </div>
+      <div class="project3">
+        <img src="images/portfolio.png" alt="" />
+        <div class="proj-desc">
+          <h2>Tonic</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt="" /></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt=""/></li>
+            <li><img src="images/css.png" alt="" /></li>
+            <li><img src="images/javaScript.png" alt=""/></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+      <div class="project4">
+        <img src="images/multi.png" alt="" />
+        <div class="proj-desc">
+          <h2>Multi-post <br>Story</h2>
+          <ul class="list1">
+            <li>CANOPY</li>
+            <li><img src="images/counter.png" alt=""/></li>
+            <li>Back End Dev</li>
+            <li><img src="images/counter.png" alt="" /></li>
+            <li>End</li>
+          </ul>
+          <p class="parag">
+            A daily selection of privately personalized reads; no accounts
+            orsign-ups required.
+          </p>
+          <ul class="proj-tech">
+            <li><img src="images/htmli.png" alt=""/></li>
+            <li><img src="images/css.png" alt=""/></li>
+            <li><img src="images/javaScript.png" alt="" /></li>
+          </ul>
+          <img class="see-proj" src="images/see.png" alt=""/>
+        </div>
+      </div>
+    </section>
+    <section class="about-sect">
+      <div class="about">
+        <h2 class="about-intro intro">About <br />Myself</h2>
+        <p class="about-pitch pitch">
+          I'm a software developer! I can help you build a product, feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hesitate to contact
```

Comment:
- [ ] Kindly select in you css file the `p` through his `class` and add this code `
font-size: large;` it going to increase the size of the paragraph.